src/Symfony/Component/Messenger/DependencyInjection/MessengerPass.php

```diff
@@ -76,15 +79,26 @@ private function registerHandlers(ContainerBuilder $container, array $busIds)
         foreach ($container->findTaggedServiceIds($this->handlerTag, true) as $serviceId => $tags) {
             foreach ($tags as $tag) {
                 if (isset($tag['bus']) && !\in_array($tag['bus'], $busIds, true)) {
-                    throw new RuntimeException(sprintf('Invalid handler service "%s": bus "%s" specified on the tag "%s" does not exist (known ones are: %s).', $serviceId, $tag['bus'], $this->handlerTag, implode(', ', $busIds)));
+                    throw new RuntimeException(sprintf(
+                        'Invalid handler service "%s": bus "%s" specified on the tag "%s" does not exist (known ones are: %s).',
+                        $serviceId,
+                        $tag['bus'],
+                        $this->handlerTag,
+                        implode(', ', $busIds)
+                    ));
                 }
 
-                $r = $container->getReflectionClass($container->getDefinition($serviceId)->getClass());
+                $className = $container->getDefinition($serviceId)->getClass();
+                $classReflector = $container->getReflectionClass($className);
+
+                if (null === $classReflector) {
```

Comment:
this is the only related change no? so could be `null === $r` and revert everything else which is not related.

Reply:
@ro0NL I thought it was clearer but it does not matter. I can revert that is not related if you consider it is not necessary.

Reply:
usually we avoid CS changes to keep the actual diff consise yes